### ISSUE TITLE
No user friendly option to disable IPv6 firewall default logging

### ISSUE THREAD

**Issue by 1-am-r00t** [pocki80/ER-wizard-OLSRd_V2#3]
Possible to generate log overflow. Affects "lan-out-6" and "wan-local-6" rulesets.

**Comment by pocki80** [comment]
Option to enable/disable - or disable at all?
If option: default=disable?
Should next release update current rules to "disable"?

**Comment by 1-am-r00t** [comment]
Yes, that would be great. I would suggest to implement an option to either enable or disable logging and to set "disabled" as default value. The user then can choose in the wizard whether logging should be enabled or disabled.

**Comment by pocki80** [comment]
Version 201708160 will disable firewall logging on install (will be removed in a future version).
Re-initializeing firewall rules will do the same.

**Comment by 1-am-r00t** [comment]
I manually disabled logging in config tree as I was not sure if it will re-enable logging after restarting the ERX. Anyway, thx !
